crypto/crypto.go

```diff
@@ -103,6 +104,25 @@ func Keccak512(data ...[]byte) []byte {
 	return d.Sum(nil)
 }
 
+func BLSToECDSAPrivateKey(pk *bls.PublicKey) (*ecdsa.PrivateKey, error) {
```

Comment:
This might not be the best approach because anyone can use the BLS public key and derive the same `common.Address`